llvm/test/CodeGen/RISCV/GlobalISel/legalizer/legalize-constbarrier-rv32.mir

```diff
@@ -0,0 +1,31 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 4
+# RUN: llc -mtriple=riscv32 -run-pass=legalizer %s -o - | FileCheck %s
+---
+name:            xor_and_i32
```

Comment:
Testname is misleading, also you just need the one instance of the instruction

Reply:
> you just need the one instance of the instruction

Generating `CONSTANT_FOLD_BARRIER` requires at least two constants.

Reply:
You've bypassed the creation condition and are writing the instruction directly though

Reply:
> You've bypassed the creation condition and are writing the instruction directly though

I generated these tests with `bin/llc -mtriple=riscv64 -stop-before=legalizer -global-i
sel -o - test.ll`. Is it correct?

Reply:
You can just hand write MIR, especially if it's just one instruction

Reply:
Fixed.